### ISSUE TITLE
Feasibility

### ISSUE THREAD

**Issue by serkolch** [salwaes/project-2#1]
@salwaes 

Hey Salwa,

I think this is a good start - there are plenty of good weather APIs that are easy enough to implement. As is, it seems like your idea is too simple, and a clone of Momentum (https://chrome.google.com/webstore/detail/momentum/laookkfknpbbblfpciffpaejjkokdgca?hl=en) and many other tools. 

We want to try and make this a bit more unique - is there another API you'd like to incorporate into this app?

**Comment by salwaes** [comment]
@serkolch 

Hey Sergey,

I was thinking of making my project a morning mood weather app and adding to the theme features to display a positive thought written by the user the evening before about a nice moment that happened during the day before, any traffic or metro problems, outfit suggestions and breakfast suggestions all that depending on the weather. So I will need a weather, a traffic/metro and a coffee shops adress list APIs but for the coffee shops list, first, I will just create my own small database of suggestions and then I will add the API later when everything else works well. What are your thoughts about these updates ? Is it still feasable ?

**Comment by serkolch** [comment]
Hey Salwa,

Looks like it might be potentially a bit much. Have you thought about which APIs this will use? It's ok to base this on a a small database first, but it should be responsive to more data eventually.

**Comment by salwaes** [comment]
Yeah true, I will remove the traffic infos and as start I will just create a small database for breakfast's meal suggestions. For the weather API I think I will use "https://www.metaweather.com/api/" and for the subway commute issues I didn't find yet one to cover all subway's news by location.
